src/template.rs

```diff
@@ -0,0 +1,56 @@
+use handlebars::{handlebars_helper, Handlebars};
+use rand::seq::SliceRandom;
+
+pub fn addhelpers(x: &mut Handlebars) {
+    handlebars_helper!(wrap: |index: usize, s: String| {
```

Comment:
this will wrap the input just once, so first line will be wrapped as expected, but 2nd line can potentially be longer than wrap index.

I think we should wrap the complete text such that all the new lines are wrapped at given index.

Reply:
this also does not handle long words and url's at the moment. golang's version of sprig handles that. you think we can include that functionality as well here?

Reply:
added long word support but not url.

Reply:
thank you. this is fine.